.github/workflows/benchmark.yml

```
@@ -70,18 +70,57 @@ jobs:
       run: |
         pytest tests
 
+  benchmarks:
+    name: Generator performance
+    runs-on: benchmark
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+
+    - uses: chia-network/actions/setup-python@main
+      name: Install Python ${{ matrix.python-version }}
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Update pip
+      run: |
+          python -m pip install --upgrade pip
+
+    - name: Set up rust
+      uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+
+    - name: setup venv
+      run: |
+        python -m venv venv
+
+    - uses: chia-network/actions/activate-venv@main
+
+    - name: Install dependencies
+      run: |
+        python -m pip install maturin
+        python -m pip install colorama
```

pip does better with fewer calls so that it can consider the whole dependency tree at one time when selecting versions.

```suggestion
        python -m pip install maturin colorama
```